style.css

```diff
@@ -435,4 +441,13 @@ header .menu-toggle {
   header .menu-toggle {
     display: none;
   }
+
+  #hero {
+    background: white url(./assets/img/Header\ bg.svg) center 0 no-repeat;
+    background-size: 100%;
+  }
```

Comment:
How should I add them? If the background image is larger than the element, it will overflow. and since we don't have same screen size all the time, I don't think we can control this. So I tried to keep it look good in different scenario. Do you have any suggestion?

I can take each part of the background separately and place them on page but I don't think this is what is expected in this project. 

I can make the container, 100vh. What do you think?

Reply:
More over, I don't think it be a good practice to modify README.md in this PR which is related to implementation of desktop version. Should commits within a given PR be relevant to it?

I had some other things to do with the project and this was one of them. I add them now but still I think readme modification shouldn't be included in this PR.